test/trento_web/controllers/v1/host_controller_test.exs

```diff
@@ -378,12 +378,36 @@ defmodule TrentoWeb.V1.HostControllerTest do
         |> json_response(:not_found)
         |> assert_schema("NotFound", api_spec)
       end
+    end
 
-      test "should forbid operation '#{saptune_operation}' if conditions are unmet", %{
-        conn: conn,
-        api_spec: api_spec
-      } do
-        %{id: host_id} = insert(:host)
+    scenarios = [
```

Comment:
nitpick: is this not an overkill? I mean, this is already tested in the policy code.
I would say that testing with `saptune_status: build(:saptune_status)` would be enough.
We don't neither test with app instances, not app instances, db instances, etc